### ISSUE TITLE
[Crashing]<Thaumcraft causing crashing>

### ISSUE THREAD

**Issue by TULDYT** [PojavLauncherTeam/PojavLauncher#5871]
### Describe the bug

I was trying to play Thaumcraft using forge and it kept crashing the game...

### The log file and images/videos

[latestlog.txt](https://github.com/user-attachments/files/16403049/latestlog.txt)

### Steps To Reproduce

```markdown
(1). Install forge for 1.12.2(14.23.5.2860).
(2). Place Thaumcraft and Baubles in the mod folder.
(3). Launch the game.
```

### Expected Behavior

The game should launch but, It keeps crashing.

### Platform

```markdown
- Device model: Techno Spark 23 4/64
- CPU architecture: Not sure
- Android version: 12
- PojavLauncher version: foxglove-20240727-ae48e8e-v3_openjdk
```

**Comment by GaucovyAero** [comment]
try searching older issues, i think somebody fixed it on the 1.7.10 version but the issue basically are the custom shaders Thaumcraft is using which Pojav does not support/fails to convert (imagine its as if you were using an old GPU from 2003), the solution is to use zink but your phone might not support it

**Comment by pbeczyuk** [comment]
> try searching older issues, i think somebody fixed it on the 1.7.10 version but the issue basically are the custom shaders Thaumcraft is using which Pojav does not support/fails to convert (imagine its as if you were using an old GPU from 2003), the solution is to use zink but your phone might not support it

Hey man, thanks so much for finding this out as it helped me alot,

Wonder if there's any way to force disable the custom shaders of Thaumcraft 6?
